index.html

```diff
@@ -19,7 +19,7 @@
         shortName: "vc-data-model",
 
         // subtitle for the spec
-        subtitle: "Expressing verifiable information on the Web",
+        subtitle: "Expressing information with verifiable provenance on the Web",
```

Comment:
```suggestion
        subtitle: "Expressing information with verifiable provenance",
```

Reply:
+1 good change, five stars, gets the job done, would buy again.